docs/src/mdsvex/li.svelte

```diff
@@ -0,0 +1,3 @@
+<li><slot /></li>
```

Comment:
If the only purpose for all of these components is to include the styles, wouldn't it be easier to just make the styles global? (especially that these components will only affect the mdsvex routes)

Reply:
Now that scoping styles has grown so much on me, definitely no. Worth it all the way

Reply:
Don't forget that this is the docs app, not the core library. If every single `<li>` will get replaced with that one instead, is it really that worth it?

Reply:
This way of styling stuff is good for two reasons: predictability and consistency. If I wanna change the style for `<li>`, I know where exactly to look for it (this is a major game changer, take it from someone who had to pull apart a crap ton of stylesheets in Innopoints that were intertwined in the most obscure of ways). Also it doesn't break the pattern of code structure needlessly. This is a low price to pay for such noticeable benefits